README.md

```diff
@@ -162,36 +155,41 @@ The Flutter Template contains:
 
 ### CD
 The [`CD`](.github/workflows/cd.yml) workflow performs the following actions:
-- Bump the build number by 1.
-- Build a signed release apk.
+- Bump the `versionCode` by 1. For details read [Version your app](https://developer.android.com/studio/publish/versioning)
+- Build a release apk (`prod` flavor).
+- Sign the apk using `apksigner`
 - Upload apk to app center.
 - Upload apk as artifact to release tag.
-- Build a signed iOS app.
-- Upload ipa to testflight.
-- Upload ipa as artifact to release tag.
 - Commit the updated version to git.
 
+| Note: It is recommended to keep your keystore and its essentials like: alias, password safe and encrypted, inside your [Github Secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+
 ### Android CD setup
 For the android CD workflow to run, we need to perform the following setup steps:
 - Follow these instructions to [generate an upload keystore](https://developer.android.com/studio/publish/app-signing#generate-key). Note down the `store password`, `key alias` and `key password`. You will need these in later steps.
 - Use `openssl` to convert the `jks` file to `Base64`.
 ```shell
-openssl base64 < flutter_template_keystore.jks | tr -d '\n' | tee flutter_template_keystore_encoded.txt
+openssl base64 < android_template_keystore.jks | tr -d '\n' | tee android_template_keystore_encoded.txt
 ```
 - Store the `base64` output on [`Github Secrets`](https://docs.github.com/en/actions/security-guides/encrypted-secrets) with the key name `KEYSTORE`.
-- Save the `store password` in github secrets with key name `RELEASE_STORE_PASSWORD`.
-- Save the `key alias` in github secrets with key name `RELEASE_KEY_ALIAS`.
-- Save the `key password` in github secrets with key name `RELEASE_KEY_PASSWORD`.
-- [Create a distribution on app center](https://docs.microsoft.com/en-us/appcenter/distribution/) and get the upload key. You can get it from from appcenter.ms/settings.
-- Save the app center upload key on github secrets with key name `APP_CENTER_TOKEN`.
+- Save the `store password` in github secrets with key name `KEYSTORE_PASSWORD`.
+- Save the `key alias` in github secrets with key name `KEY_ALIAS`.
+- Save the `key password` in github secrets with key name `KEY_PASSWORD`.
+- [Create a distribution on app center](https://docs.microsoft.com/en-us/appcenter/distribution/) and get the upload key. You can get it from from [Settings](https://appcenter.ms/settings).
+- Save the app center upload key on github secrets with key name `APP_CENTER_USER_API_TOKEN`.
+- Save the group name as `GROUP_NAME` inside github secrets. Example: `Testers`
+- Save the organisation name as `ORG_NAME` inside github secrets. Example: `Wednesday Solutions`
+- Save the app name as `APP_NAME` inside github secrets. Example: `Android Template`
+
+| Caution: Respect the of the value inside secrets or else AppCenter APIs might have problems looking for your app
```

Comment:
Something seems to be incorrect here.

Reply:
Resolved by changing the `I` to `>`, I actually intended to write a block